src/index.common.js

```diff
@@ -9,9 +9,23 @@ const ngModuleName = 'formly';
 const angular = require('./angular-fix');
 const ngModule = angular.module(ngModuleName, []);
 
-require('./providers')(ngModule);
-require('./services')(ngModule);
-require('./directives')(ngModule);
-require('./run')(ngModule);
+ngModule.constant('formlyApiCheck', require('./providers/formlyApiCheck'));
```

Comment:
Using ```require``` vs ```import``` was simply, as you alluded to, to avoid boilerplate.  In the case of this file, I took a shortcut to be able to inline the requires to register in the angular module.  However, I agree that, for consistency, ES6 module syntax should probably be used throughout.